packages/sil-governance/src/config/index.ts

```diff
@@ -25,9 +25,13 @@ interface WindowWithConfig extends Window {
   readonly developmentConfig: Config;
 }
 
+function getWindowWithConfig(): WindowWithConfig {
```

Comment:
if you will implement this function in `ui-logic` package you will not need this implementation.